### ISSUE TITLE
HubAi dont return image in production with @cf/bytedance/stable-diffusion-xl-lightning (in dev it works)

### ISSUE THREAD

**Issue by samulefevre** [nuxt-hub/core#242]
```ts
const res = await hubAI().run('@cf/bytedance/stable-diffusion-xl-lightning', {
   prompt,
}) // return Uint8Array
// seems like in production it don't return the image (in dev i get image)

const blob = new Blob([res])

// then save image with hubBlob() with blob
// in dev i see image in blobs, in prod i got empty image(15B)


```

<img width="1351" alt="Capture d’écran 2024-08-27 à 13 14 07" src="https://github.com/user-attachments/assets/ee1cfb84-ae4a-4604-822e-2d761c62bc7d">

<img width="964" alt="Capture d’écran 2024-08-27 à 15 49 54" src="https://github.com/user-attachments/assets/f6f5762c-a9cb-4dff-acf0-739335093984">

<img width="993" alt="Capture d’écran 2024-08-27 à 15 53 50" src="https://github.com/user-attachments/assets/38c6d245-6509-4e49-ae58-53ff9e124204">

But why in cloudflare account, it say i use @cf/baai/bge-small-en-v1.5

(it works in dev, i got image but in prod no image)

i tried with other model : @cf/stabilityai/stable-diffusion-xl-base-1.0.   same result works in dev but not working in production, what the hell is happening ?

**Comment by atinux** [comment]
Do you have a small reproduction I can investigate @samulefevre ?

On solution it to make sure to transform as blob only when it's not one:

```ts
let blob = await hubAI().run('@cf/bytedance/stable-diffusion-xl-lightning', {
   prompt,
})
if (blob instanceof Uint8Array) {
  blob = new Blob([blob])
}
```

I also have some `@cf/baai/bge-small` in my account on Cloudflare but I don't think it's related as beta models are not displayed in their dashboard.

**Comment by samulefevre** [comment]
It works with your solution, thank you !

**Comment by atinux** [comment]
Pushed a fix in [v0.7.7](https://github.com/nuxt-hub/core/releases/tag/v0.7.7) so you don't have to do this anymore normally 😊

**Comment by samulefevre** [comment]
Nice ! Thank you
